content/blog/hpe-greenlake-for-private-cloud-enterprise-scaling-and-orchestrating-modern-applications-for-the-enterprise.md

```diff
@@ -0,0 +1,95 @@
+
+**Kubernetes (K8s) on HPE GreenLake Private Cloud Enterprise**
+HPE GreenLake for Private Cloud Enterprise provides Kubernetes deployment via its container services. Notably, it's a CNCF-compliant K8s distribution, ensuring adherence to industry standards.
+
+**Key Points:**
+
+1. On-premises Kubernetes: It is designed for on-premises deployments with the ability to scale based on business needs.
+2. Ready to use: The platform is pre-configured, reducing setup time.
+3. Performance and security: It provides consistent performance and security for operations.
+4. Quick Kubernetes start: The platform features one-click provisioning that gets Kubernetes operations running fast.
+5. Central Kubernetes management: Its management console allows for easy control over Kubernetes clusters.
+6. Pay-as-you-go pricing: It uses a flexible pricing model where businesses pay for what they use.
+
+Containers service stands out by delivering the security, efficiency, and cost-effectiveness modern businesses seek, maximizing resource use. In tandem with Kubernetes, containers service provides a smooth and managed infrastructure. Enhanced by HPE Ezmeral Runtime Enterprise, it gives businesses a refined platform for deploying applications, ensuring scalability, and streamlined operations.
+
+**Under the hood – A fungible infrastructure resource pool**
+As discussed in [Part 1](https://developer.hpe.com/blog/hpe-greenlake-for-private-cloud-enterprise--Exploring-a-flexible-infrastructure-resource-pool/) of this blog series, HPE GreenLake for Private Cloud Enterprise integrates an adaptable infrastructure that leverages the strengths of its infrastructure resource pool and detailed capacity planning. This approach not only addresses present business requirements but also anticipates future needs. Within HPE GreenLake for Private Cloud Enterprise, the resource pool offers flexibility, letting businesses choose container deployments on bare metal, virtual machines, or a combination of both, based on their specific demands. 
+Diving into the heart of HPE GreenLake for Private Cloud Enterprise: Containers , we encounter two primary deployment models for Kubernetes: on virtual machines and on bare metal.
+Kubernetes on bare metal:
+•	Tailored for expansive, consistent deployments where performance is paramount.
+•	Efficiency takes center stage, devoid of VM-associated costs.
+•	Simplified management structure cuts down on overhead
+•	Perfect for business-critical applications or those where speed and responsiveness are crucial, like financial or imaging apps.
+
+Kubernetes on virtual machines offers:
```

Comment:
Add a line space (or insert 4 spaces) just after the sentence "Kubernetes on virtual machines offers:" in order to separate the bullet list items.